### PR TITLE
Settings Page

### PR DESCRIPTION
**Pull request by adamjhc**

### CODE REVIEW

File: src/components/SettingItem.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+const Subtitle = styled.div`
+  color: var(--grey-color);
+  font-size: 1.4rem;
+`
+
+interface Props {
+  title: string
+  subtitle?: string
+  href?: string
```
Comment (timothy-ch-cheung) [inline src/components/SettingItem.tsx:16]: Should href be optional?

File: src/pages/settings/index.tsx [hunk 2]
```diff
@@ -1,6 +1,26 @@
 import Head from "next/head"
+import { styled } from "linaria/react"
 
 import TopBar from "../../components/TopBar"
+import SettingItem from "../../components/SettingItem"
+
+const Main = styled.main`
```
Comment (pavsidhu) [inline src/pages/settings/index.tsx:7]: The entire window except the tab bar is contained within a main so don't wanna nest a main within a main, for this I think an `<article />` is recommended

File: src/components/SettingItem.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { styled } from "linaria/react"
+
+const Title = styled.div`
+  font-size: 1.8rem;
+  font-weight: 500;
```
Comment (pavsidhu) [inline src/components/SettingItem.tsx:5]: Would recommend adding a 4px margin between the title and subtitle

File: src/pages/settings/index.tsx [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-s);
```
Comment (pavsidhu) [inline src/pages/settings/index.tsx:10]: flex gap isn't fully supported yet, it's really annoying: https://caniuse.com/?search=flex%20gap

In this case I've been using a bit of a hack as gap is widely supported for grids:

```css
display: grid;
grid-auto-flow: row; /* This sets the children to implicit rows in the grid */
gap: 8px;
```

File: src/pages/settings/index.tsx [hunk 5]
```diff
@@ -1,6 +1,26 @@
...
+
+  margin: var(--space-s) var(--space-m) 0;
+
+  @media (min-width: 1200px) {
+    max-width: 800px;
+  }
+`
+
+const Separator = styled.hr`
```
Comment (pavsidhu) [inline src/pages/settings/index.tsx:19]: I forgot hr tags were a thing, noice

File: src/components/SettingItem.tsx (hunk 1 above)
Comment (pavsidhu) [reply src/components/SettingItem.tsx:16]: Yeah just checked this is fine, for example, there's a version number in the settings which shouldn't be clickable

Comment (pavsidhu) [reply src/components/SettingItem.tsx:16]: Maybe this is bad UX but it's minor, something to look at in the future

**Review by pavsidhu — CHANGES_REQUESTED** [review]

File: src/components/SettingItem.tsx (hunk 1 above)
Comment (adamjhc) [reply src/components/SettingItem.tsx:16]: Yeah thought about adding a different tag for settings that aren't clickable but when href is null it doesn't get added to the `<a>`  tag and so isn't clickable anyway

Comment (pavsidhu) [reply src/components/SettingItem.tsx:17]: That's a good point actually if there's no href we shouldn't render an anchor tag anyway, I think that would confuse screen readers could do to simply a better solution:

```jsx
const Container = styled.a

return (
  <Container as={href ? "a" : "div"}>
    ...
  </Container>
)
```

**Review by timothy-ch-cheung — CHANGES_REQUESTED** [review]

File: src/styles/global.css [hunk 6]
```diff
@@ -6,8 +6,8 @@
   --blue-color: #caeaf8;
   --purple-color: #e4d0f2;
   --green-color: #cef8ca;
-  --grey-color: #8b8b8b;
-  --light-grey-color: #ececec;
+  --grey-color: #757575;
+  --light-grey-color: #eaeaea;
```
Comment (adamjhc) [inline src/styles/global.css]: @pavsidhu think you made this change, just wanna make sure this change doesn't get lost if we go back on it

**Review by pavsidhu — APPROVED** [review]

**Pull request merged**